CHANGELOG.md

```diff
@@ -18,6 +20,214 @@
+- Capacity for TLS traffic [#3855](https://github.com/Kong/kong/pull/3855)
+- Certificate Authority management [#3906](https://github.com/Kong/kong/pull/3906), [#3861](https://github.com/Kong/kong/pull/3861),
+- `transparent` listeners allow setting up a service mesh with `ip-tables` [#3884](https://github.com/Kong/kong/pull/3884)
+- Plugins' `run_on` field to control how they behave on a Service Mesh
+  environment [#3930](https://github.com/Kong/kong/pull/3930), [#4066](https://github.com/Kong/kong/pull/4066)
+- There is a new phase called `preread`. This is where stream traffic routing is done.
+- Turned on TLSv1.3 in our templates to workaround a bug in OpenSSL [#4046](https://github.com/Kong/kong/pull/4046)
+
+##### Core
+
+- :fireworks: Routes can be named [#3764](https://github.com/Kong/kong/pull/3764)
+- :fireworks: Implemented TTL support in the new DAO & Admin API [#3603](https://github.com/Kong/kong/pull/3603), [#3638](https://github.com/Kong/kong/pull/3638)
+- Schema improvements:
+  - Subschemas [#3630](https://github.com/Kong/kong/pull/3630)
+  - New entity validators: `distinct`, `ne`, `is_regex`, `contains`, `gt`
+  - Entity checks are only run when necessary [#3848](https://github.com/Kong/kong/pull/3848)
+  - Conditional validators can mark fields as `required` [6d1707c4](https://github.com/Kong/kong/commit/6d1707c4)
+  - Partial updates of record fields [05adc40f](https://github.com/Kong/kong/commit/05adc40f)
+  - ... and many other quality of life improvements
+- PDK improvements:
+  - New `kong.node` module [#3826](https://github.com/Kong/kong/pulls/3826)
+  - New `kong.response.get_path_with_query` module [#3842](https://github.com/Kong/kong/pull/3842)
+  - PDK getters and setters for Service, Route, Consumer & Credential [#3916](https://github.com/Kong/kong/pull/3916)
+  - `kong.response.get_source` returns `error` on errors [#4006](https://github.com/Kong/kong/pull/4006)
+  - `kong.response.exit` can be used in `header_filter` phase, but only without
+    a body [#4039](https://github.com/Kong/kong/pull/4039)
+- New config options:
+  - `dns_valid_ttl` [#3730](https://github.com/Kong/kong/pull/3730)
+  - `pg_timeout` [#3808](https://github.com/Kong/kong/pull/3808)
+  - `upstream_keepalive` can be disabled when set to 0 (thanks,
+    [@pryorda](https://github.com/pryorda)!) [#3716](https://github.com/Kong/kong/pull/3716)
+  - The new `transparent` suffix also applies to the `proxy_listen` directive
+- Cluster-wide mutex [#3685](https://github.com/Kong/kong/pull/3685)
+- Add multipart support to Admin API: [#3776](https://github.com/Kong/kong/pull/3776)
+- Improved plugins iterator speed [#3794](https://github.com/Kong/kong/pull/3794)
+- Add support for HTTPS in active healthchecks [#3815](https://github.com/Kong/kong/pull/3815)
+
+##### Plugins
+
+- New options `no_api`, `no_route` & `no_service` & `no_consumer` to limit
```

Comment:
Note that this is for 0.15 only and was replaced by typedefs in 1.0

Reply:
I have added a new `#### Plugins` section to the `1.0.0` "Breaking changes" section to indicate that they are typedefs there.

Reply:
the whole new schema format is a `### Plugins` breaking change for 1.0 really

Reply:
You are absolutely right. I have changed it to be all `### Plugins`